src/js/safari/safari_shim.js

```diff
@@ -350,3 +350,48 @@ export function shimAudioContext(window) {
   }
   window.AudioContext = window.webkitAudioContext;
 }
+
+// Note: Make sure to call this ahead of shimCallbacksAPI.
+export function shimParameterlessSetLocalDescription(window, browserDetails) {
+  if (!(window.RTCPeerConnection && window.RTCPeerConnection.prototype)) {
+    return;
+  }
+  const nativeSetLocalDescription =
+      window.RTCPeerConnection.prototype.setLocalDescription;
+  if (!nativeSetLocalDescription || nativeSetLocalDescription.length === 0) {
+    return;
+  }
+  window.RTCPeerConnection.prototype.setLocalDescription =
+    function setLocalDescription() {
+      let desc = arguments[0] || {};
+      if (typeof desc != "object" || (desc.type && desc.sdp)) {
```

Comment:
```suggestion
      if (typeof desc !== "object" || (desc.type && desc.sdp)) {
```
needs extra check for rollback?

Reply:
Good catch on `"rollback"`!

I'll yield to the prevailing pattern in this module on !== and ===